app/Http/Routes/hotel_routes.php

```diff
@@ -23,3 +23,19 @@
     'as' => 'hotel.logout',
     'uses' => 'Hotel\AuthController@getLogout'
 ]);
+
+/**
+ * Route for hotel admin get to edit hotel page
+ */
+Route::get('hotel/edit', [
```

Comment:
- Route name should be `hotel/profile`. 
- create the route group for hotel and set prefix `hotel`. See here http://laravel.com/docs/5.1/routing#route-groups